framework/src/media/utils/MediaUtils.cpp

```diff
@@ -511,6 +558,71 @@ bool header_parsing(unsigned char *buffer, unsigned int bufferSize, audio_type_t
 	return true;
 }
 
+#ifdef CONFIG_CONTAINER_MPEG2TS
+bool ts_parsing(unsigned char *buffer, unsigned int bufferSize, audio_type_t *audioType, unsigned int *channel, unsigned int *sampleRate, audio_format_type_t *pcmFormat)
```

Comment:
actually header_parsing is added here because there's chance that reveal header in application in future to get id3 tag values, and so on.
Is ts has same chance?

Reply:
header_parsing call ts_parsing to demux and get ES data, for mpeg2 transport stream.
application always call header_parsing, not ts_parsing.